draft-ietf-quic-transport.md

```diff
@@ -3667,12 +3667,12 @@ acknowledgements.
 
 An endpoint measures the delays intentionally introduced between the time the
 packet with the largest packet number is received and the time an acknowledgment
-is sent.  The endpoint encodes this delay in the Ack Delay field of an ACK
+is sent.  The endpoint encodes this delay in the ACK Delay field of an ACK
```

Comment:
I think that this is good.

However, some of the other uses here refer to a concept and that is tricky.  We don't really define "ACK delay" as a concept and we probably should.

Reply:
The principle that I thought was used elsewhere, and which I tried to apply here, was to call it "ACK Delay" when talking about the frame field, and "ACK delay" otherwise.

Reply:
We shouldn't use "ACK delay" without defining it, and if we do, I would suggest using "acknowledgement delay" instead.

Reply:
Applying Jana's change.